### PR TITLE
Update sv.json

### PR DESCRIPTION
**Pull request by DewGew**

### CODE REVIEW

File: ui/src/i18n/sv.json [hunk 1]
```diff
@@ -42,27 +42,10 @@
...
-    "backup.message_backup_help_two": "Plugins not hosted on npm, or plugins installed directly from GitHub will not be backed up or restored.",
-    "backup.message_backup_restored": "Backup Archive Restored",
-    "backup.message_backup_warning": "Backup archives contain sensative information and should not be shared with others.",
-    "backup.message_restore_failed": "Restore Failed",
-    "backup.message_restore_help_one": "Restore a backup archive you previously made using Homebridge Config UI X. Your Homebridge config, cached accessories, HomeKit pairings and Homebridge Config UI X user accounts will be restored. Any Homebridge plugins you had installed will also be downloaded from the npm registry.",
-    "backup.message_restore_help_two": "You will need to login to Homebridge Config UI X again after doing a restore using the credentials from the estored instance.",
-    "backup.message_restore_warning": "Restoring from a backup will irreversibly overwrite the current Homebridge configuration.",
-    "backup.title_backup": "Backup",
-    "backup.title_backup_restore": "Backup / Restore",
-    "backup.title_restore": "Restore",
```
Comment (oznu) [inline ui/src/i18n/sv.json]: Thanks @DewGew for the PR.

Would you mind rebasing your branch to include these recently added messages?

Comment (DewGew) [reply ui/src/i18n/sv.json]: I noticed that to :)

Comment (DewGew) [reply ui/src/i18n/sv.json]: Fixed

**Comment by oznu** [comment]
Thank you!

**Pull request merged**